packages/lime-system/files/usr/lib/lua/lime/network.lua

```diff
@@ -238,7 +248,7 @@ function network.configure()
 		if utils.isModuleAvailable(protoModule) then
 			local proto = require(protoModule)
 			xpcall(function() proto.configure(utils.split(protocol, network.protoParamsSeparator)) end,
-			       function(errmsg) print(errmsg) ; print(debug.traceback()) end)
+				   function(errmsg) print(errmsg) ; print(debug.traceback()) end)
```

Comment:
these lines are just due to using `%s/    /^I/g` (aka replace whitespaces with tabs), I guess it doesn't hurt to unify the code